### DISCUSSION TITLE
Change background when alarm is triggered

### DISCUSSION THREAD

**Discussion by timofort** [BlackyHawky/Clock#80] [Ideas]
Hello and thank you for proposing new version of Clock.

In 1.2 version, alarm started with our own phone lockscreen as background. Not anymore, in 2.5 version for exemple.

Is it possible to put back this feature in next release(s), and extend it to timer in addition ?

Thanks.

Timofort

**Comment by BlackyHawky** [comment]
Indeed, I chose a dark color so as not to be dazzled when we wake up and have to stop (or snooze) the alarm.

Perhaps defining a transparent color will answer your request. I'll have to test it. (I've never used version 1.2 except for a few tests.)

Perhaps we can also implement the color picker so that users can choose their own color.

I'll look into it soon.

**Reply by timofort** [reply]
Thank you ;)

**Reply by BlackyHawky** [reply]
Taken into account with issue #85
